src/app/[locale]/(default)/events/[id]/page.tsx

```diff
@@ -0,0 +1,97 @@
+import { getTranslations, setRequestLocale } from 'next-intl/server';
+
+import { Avatar, AvatarImage } from '@/components/ui/Avatar';
+import { Badge } from '@/components/ui/Badge';
+import { Separator } from '@/components/ui/Separator';
+// TODO: Must be replaced with actual events
+import { events } from '@/mock-data/events';
+import { CalendarIcon, MapPinIcon } from 'lucide-react';
+import { notFound } from 'next/navigation';
+
+export async function generateMetadata({
```

Comment:
This generatemetadata should probably be moved to the root events page, and here we can get the name of the event and insert as the title instead

Reply:
Using `generateMetadata` to add event title name now instead